code/apps/Managed Software Center/Managed Software Center/en.lproj/MainMenu.xib

```diff
@@ -439,10 +442,10 @@
                         <autoresizingMask key="autoresizingMask" widthSizable="YES" heightSizable="YES"/>
                         <clipView key="contentView" id="rwq-IA-IvG">
                             <rect key="frame" x="1" y="1" width="512" height="447"/>
-                            <autoresizingMask key="autoresizingMask" widthSizable="YES" heightSizable="YES"/>
+                            <autoresizingMask key="autoresizingMask"/>
                             <subviews>
                                 <tableView focusRingType="none" verticalHuggingPriority="750" allowsExpansionToolTips="YES" columnAutoresizingStyle="lastColumnOnly" alternatingRowBackgroundColors="YES" columnReordering="NO" columnResizing="NO" autosaveColumns="NO" typeSelect="NO" id="UAy-M3-yRr">
-                                    <rect key="frame" x="0.0" y="0.0" width="512" height="17"/>
+                                    <rect key="frame" x="0.0" y="0.0" width="512" height="447"/>
```

Comment:
I just flipped it back in https://github.com/munki/munki/pull/736/commits/1198e6c56a5511835d757ed7f7969d1dd2cd6f11.

I suspect there's some property here that Xcode wants to keep snapping back to a different default, because when I tried to find the element within the Log window within the IB interface, without modifying any actual properties, MainMenu.xib had set something else to 447 too.